src/components/molecules/ContentSearchbar.stories.js

```diff
@@ -0,0 +1,34 @@
+import { storiesOf } from "@storybook/vue";
+import { text } from "@storybook/addon-knobs";
+import { select } from "@storybook/addon-knobs";
+
+import ContentSearchbar from "@components/molecules/ContentSearchbar";
+
+const template = `
+	<content-searchbar
+		v-model.lazy="searchQuery"
+		:class="
+			!activateTransition ? 'first-search__searchbar' : 'content__searchbar'
+		"
+		placeholder="Lernstore durchsuchen"
```

Comment:
```suggestion
		:placeholder="placeholder"
```
It's a good idea to actually use the knobs you defined